pkg/rules/manager.go

```diff
@@ -97,12 +97,14 @@ func ActiveAlertsToProto(s storepb.PartialResponseStrategy, a *rules.AlertingRul
 	active := a.ActiveAlerts()
 	ret := make([]*rulespb.AlertInstance, len(active))
 	for i, ruleAlert := range active {
+		// https://github.com/gogo/protobuf/issues/519
```

Comment:
Let's keep comments consistent with the other workarounds as well.
```suggestion
		// UTC needed due to https://github.com/gogo/protobuf/issues/519.
```

Reply:
Hello,
I updated the comment (+ another similar one found in the codebase)